### PR TITLE
PLANET-6114 Add commitlint CI job

### PR DESCRIPTION
**Pull request by comzeradd**
Ref: https://jira.greenpeace.org/browse/PLANET-6114

---

Same as planet4-master-theme/pull/1386

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "@babel/plugin-proposal-nullish-coalescing-operator": "*",
     "@babel/plugin-proposal-optional-chaining": "*",
+    "@commitlint/cli": "^12.1.4",
```
Comment (lithrel) [inline package.json:20]: Maybe you should put this version in master-theme too ? (it's 12.1.1 in the other PR)

**Review by lithrel — APPROVED** [review]
:+1:

File: package.json (hunk 1 above)
Comment (comzeradd) [reply package.json:20]: Oh right. They release a new version since I first tried this out :grin:

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [mars](https://www-dev.greenpeace.org/test-mars)

:watch: 2021.05.27 10:10:22

**Pull request merged**